src/set_theory/zfc/basic.lean

```diff
@@ -44,6 +44,8 @@ Then the rest is usual set theory.
   function `x → y`. That is, each member of `x` is related by the ZFC set to exactly one member of
   `y`.
 * `Set.funs`: ZFC set of ZFC functions `x → y`.
+* `Set.hereditarily p x`: Predicate that every set in the transitive closure of `x` has property
```

Comment:
Is "transitive closure" commonly used to describe this set? Otherwise, I would suggest
```suggestion
* `Set.hereditarily p x`: Predicate that every set in the transitive closure of `x` by set membership has property
```

Reply:
I think it is. See https://ncatlab.org/nlab/show/transitive+closure and https://en.wikipedia.org/wiki/Transitive_set#Transitive_closure.

Reply:
It would be nice to prove `hereditarily p x ↔ ∀ y, relation.refl_trans_gen (∈) y x → p y`.